components/app.js

```diff
@@ -0,0 +1,88 @@
+function getAverage(grades){
+  var total = 0
+  for(var i = 0; i < grades.length; i++){
+    total = total += grades[i].grade
+    var avg = total/grades.length
```

Comment:
It seems like extra work to calculate the average at every step, rather than once at the end.